src/Components/ADT3DSceneBuilder/ADT3DSceneBuilder.styles.ts

```diff
@@ -39,7 +39,7 @@ export const getStyles = (
                     alignItems: 'center',
                     position: 'absolute',
                     right: 0,
-                    top: BUILDER_HEADER_TOP_OFFSET,
+                    top: BUILDER_CONTROLS_TOP_OFFSET,
```

Comment:
I still don't see why this is changing just to add borders and horizontal padding on the left panel items. What is giving us the vertical changes?

Reply:
And it needs to be local so the right border goes fully to the top?